gulpfile.js

```diff
@@ -64,114 +68,112 @@ const unminifiedRuntimeTarget = 'dist/amp.js';
 const unminified3pTarget = 'dist.3p/current/integration.js';
 
 // Each extension and version must be listed individually here.
-declareExtension('amp-3q-player', '0.1', {hasCss: false});
```

Comment:
Removing the option for `hasCss` seems disconnected from the goal of adding TypeScript support. Are these changes necessary or can we ensure the arity of the declareExtension function is always matching?

Reply:
This code shouldn't be hot path, but we should verify the impact of mismatching arity.

This isn't a blocker by any means, just good for us to know.